src/__tests__/ceramic_integration.test.ts

```diff
@@ -113,6 +115,29 @@ async function makeGanache(startTime: Date, port: number): Promise<Ganache.Serve
   return ganacheServer
 }
 
+class FauxAnchorLauncher {
+  port: number
+  server: Server
+  start(port: number) {
+    const app = express()
+    app.all('/', (req, res) => {
+      res.send({status: 'success'})
+    })
+    this.server = app.listen(port, () => {
+      console.log(`Listening on port ${port}`)
+    })
+  }
+  stop() {
+    this.server.close()
```

Comment:
small nitpick: close takes a callback so it returns before everything is nicely shut down. Not waiting shouldn't be too much of a big deal in tests  but to ensure tests run smoothly you can replace this with: 
```
  async stop() {
    return new Promise((resolve) => this.server.close(resolve))
  }
  ```
  
  And then await on line 392